tests/unit/ci.sh

```diff
@@ -1,6 +1,7 @@
 function main(){
   set -ex
   speech_ci_path=`pwd`
+  pip install ffmpeg flatten_dict ffmpy
```

Comment:
`test_audiotools.sh` will install these pkg, I think this don't need to add it twice?